### DISCUSSION TITLE
Sync shows files as deleted in view but not in edit in GitBook

### DISCUSSION THREAD

**Discussion by AnjaniGourisaria** [GitbookIO/community#293] [0. General bugs and questions]
When syncing work from GitBook to GitLab,  files appear as "deleted" in the view but are still present in the edit mode on GitBook. This discrepancy causes confusion and can lead to potential data loss or issues with version control.
Before edit:
![image](https://github.com/GitbookIO/community/assets/82677043/6e99c5e3-8910-4081-bd62-25c8a2fd5457)
After edit:
![image](https://github.com/GitbookIO/community/assets/82677043/422e5d37-6697-4a46-9652-f4d24a16dc8c)
In GitLab File:
![image](https://github.com/GitbookIO/community/assets/82677043/d07a7dd6-075d-490f-9ffe-7b8731d3df18)

**Comment by aikoii** [comment]
Hi @AnjaniGourisaria! So sorry you are facing this issue! Could you send me the links to the page where your files _appear as "deleted"_ and the change request where _they are still present_? You can send them to [support@gitbook.com](mailto:support@gitbook.com) 🙏.
